packages/content-fetch/src/request_handler.ts

```diff
@@ -138,7 +138,7 @@ export const contentFetchRequestHandler: RequestHandler = async (req, res) => {
     console.log('cacheFetchResult result', cacheResult)
 
     const jobs = await queueSavePageJob(savePageJobs)
-    console.log('save-page jobs queued', jobs.length)
+    console.log('save-page jobs queued', { jobs })
```

Comment:
not sure if we need to log everything here if we want to reduce the log size

Reply:
Yeah I'll remove this now